docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -121,11 +117,58 @@ Training is an automatic process by which Model Builder teaches your model how t
 
 Because Model Builder uses automated machine learning (AutoML), it does not require any input or tuning from you during training.
 
+### How long should I train for?
+
+Model Builder uses AutoML to explore multiple models to find you the best performing model.
+
+Longer training periods allow AutoML to explore more models with a wider range of settings.
+
+The table below summarizes the average time taken to get good performance for a suite of example datasets.
+
+|Dataset size|Average time to train|
+|------------|---------------------|
+|0 - 10 MB|10 sec|
+|10 - 100 MB|10 min|
+|100 - 500 MB|30 min|
+|500 - 1 GB|60 min|
+|1 GB+|3+ hours|
+
```

Comment:
It might be good to mention that the exact training time is a based in a few things like:
The number of features / columns being used to predict
The type of columns i.e. text vs. numeric
The chosen ML task (e.g. regression vs. classification)
User's machine

Also this is specifically for local training so should specify that.